plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js

```diff
@@ -3691,6 +3703,18 @@ test.describe('Data API tests', () => {
 						"weight_unit": "kg",
 						"states": expect.arrayContaining([])
 					},
+					{
+						"code": "XK",
+						"name": "Euro",
+						"currency_code": "EUR",
+						"currency_pos": "right_space",
+						"decimal_sep": ",",
+						"dimension_unit": "cm",
+						"num_decimals": 2,
+						"thousand_sep": " ",
+						"weight_unit": "kg",
+						"states": []
+					},
```

Comment:
I think I understand what's going on here: in `continents.php` we're adding `XK` at the end of the Europe list, but here we're adding it as the second last item. That way the API response object will mismatch the expected one.

Reply:
Gave this a try - I reordered xk so it is always added last to the list, but it didn't do the trick.

Reply:
Seems like it helped - there's just one failure now (failing 4 times)!

I think you only need to add the new country in the expected list of countries in this other failing test, too:

https://github.com/woocommerce/woocommerce/blob/5853a8d7845351eadbc02edac5e1cdbdffc07421/plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js#L2066